### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by zalseryani** [opensearch-project/cross-cluster-replication#898]
**What is the bug?**
- After creating a cross-cluster replication for one index, I started getting this error knowing that the replication is happening but why this error is occurring ?

```
Encountered a failure while executing in org.opensearch.replication.action.changes.GetChangesRequest
```

![image](https://github.com/opensearch-project/cross-cluster-replication/assets/49269877/91f50c85-0ed2-4af2-8329-bed7b005fbe0)

**How can one reproduce the bug?**
- just create cross-site replication between 2 k8s clusters where the service of opensearch is exposed with LoadBalancer on GKE 

**What is the expected behavior?**
- replication without log errors. 

**What is your host/environment?**
- k8s cluster on GKE v1.24.11
- helm chart version --> 2.12.0
- AppVersion --> 2.7.0

**Do you have any additional context?**
- Also I noticed that opensearch takes more time to update/flush its indices in order to have the data available when executing 

```
GET _cat/indices/leader-test-follower?v
```

- How to decrease the flush time of opensearch ? and when it is being done ? 

Thanks in advance.

**Comment by soosinha** [comment]
`OpenSearchTimeoutException` is an expected exception thrown when no new operations are available in the leader cluster. The `GetChangesAction` waits for 1 minute for the global checkpoint to advance before throwing the exception.

> Also I noticed that opensearch takes more time to update/flush its indices in order to have the data available when executing

Can you provide details whether this is regarding the indices on leader or the follower? And are you are looking at the doc count to infer that the indices are updated ?

**Comment by zalseryani** [comment]
@soosinha 

Yes, I am checking the doc count, knowing that the index refresh interval is 1 second.

**Comment by hm2thr33** [comment]
Hi,
have the same issue with OpenSearch 2.8.0

@soosinha how can i increase the parameter `GetChangesAction`?

and it seems to me that this option is being ignored:

```
    "plugins": {
      "replication": {
        "follower": {
          "metadata_sync_interval": "60s"
        }
```
because the index that I synchronize is updated every ~10 seconds
